src/Symfony/Component/Cache/Adapter/AbstractAdapter.php

```diff
@@ -108,16 +108,12 @@ function ($deferred, $namespace, &$expiredIds) use ($getId) {
      */
     public static function createSystemCache($namespace, $defaultLifetime, $version, $directory, LoggerInterface $logger = null)
     {
-        if (null === self::$apcuSupported) {
-            self::$apcuSupported = ApcuAdapter::isSupported();
+        $opcache = new PhpFilesAdapter($namespace, $defaultLifetime, $directory, true);
```

Comment:
no `PhpFilesAdapter::isSupported();` falling back to `FilesystemAdapter`?

Reply:
Nope, unconditional is simpler and we don't care if you don't have opcache :)

Reply:
but, if the user does not have opcache or apcu, then you get no caching at all?   possible hard to debug? up to the user to manually configure?

Reply:
just looked at it does fallback to filesystem in 4.1 FYI
https://github.com/symfony/symfony/blob/a57ce22ab5ceba15f8606b8955d715f232c279f6/src/Symfony/Component/Cache/Adapter/AbstractAdapter.php#L89-L123

Reply:
if you don't have opcache, you don't care about that

Reply:
no opcache means you will still have the cache - but php will parse the files all the time

Reply:
ah yes! gotcha.